client/app/dedicatedCloud/resource/upgrade/index.js

```diff
@@ -0,0 +1,23 @@
+import {
+  controller,
+  controllerName,
+} from './controller';
+
+import state from './state';
+
+const MODULE_NAME = 'resourceUpgrade';
+
+angular
+  .module(MODULE_NAME, [
+    'oui',
+    'pascalprecht.translate',
+    'ui.router',
+  ])
+  .controller(controllerName, controller)
```

Comment:
I'm not sure the file containing the `class` should also be in charge of the `angular` naming. Also as you gave a generic name, won't it conflicts with other modules if we have another `resourceUpgrade` controller ?

Reply:
1. This was debated before with @frenautvh and we decided it was the best course of action, but if you think otherwise, I'm all hears for your arguments :)
2. I really don't know, I asked @cbourgois who told me it was fine and that is was better than `dedicatedCloudResourceUpgrade`. I think the whole point is to say that `resourceUpgrade` could be used at more than one place, so it doesn't make sense to name it `dedicatedCloud`